cdc/kv/client.go

```diff
@@ -1134,6 +1134,14 @@ func (s *eventFeedSession) singleEventFeed(
 							CRTs:    entry.CommitTs,
 						},
 					}
+
+					if atomic.LoadUint32(&initialized) == 1 && entry.CommitTs <= lastResolvedTs {
```

Comment:
```suggestion
					if entry.CommitTs <= lastResolvedTs && atomic.LoadUint32(&initialized) == 1 {
```

Please check resolved ts first so that we can avoid atomic load every time.